### ISSUE TITLE
Both JPG and PNG files cause failures in /node_modules/gifsicle/vendor/gifsicle

### ISSUE THREAD

**Issue by ammarjabakji** [wprig/wprig#500]
Adding  JPG and PNG files to /dev/images causes errors with /node_modules/gifsicle/vendor/gifsicle

Error output
```

[08:37:23] 'images' errored after 38 ms
[08:37:23] Error in plugin "gulp-image"
Message:
    spawn /Applications/XAMPP/xamppfiles/htdocs/wp/wp-content/themes/wmg/node_modules/gifsicle/vendor/gifsicle ENOENT
Details:
    errno: ENOENT
    code: ENOENT
    syscall: spawn /Applications/XAMPP/xamppfiles/htdocs/wp/wp-content/themes/wmg/node_modules/gifsicle/vendor/gifsicle
    path: /Applications/XAMPP/xamppfiles/htdocs/wp/wp-content/themes/wmg/node_modules/gifsicle/vendor/gifsicle
    spawnargs: --optimize,--output,/private/var/folders/gx/kvm0lgq157q2bwjwq_jd18xm0000gq/T/f48b4761-8eff-4171-8857-edd5de0b3887,/private/var/folders/gx/kvm0lgq157q2bwjwq_jd18xm0000gq/T/5d01229f-0c79-4d8f-8e3b-a1398c1dc551
    killed: false
    stdout: 
    stderr: 
    failed: true
    signal: null
    cmd: /Applications/XAMPP/xamppfiles/htdocs/wp/wp-content/themes/wmg/node_modules/gifsicle/vendor/gifsicle --optimize --output /private/var/folders/gx/kvm0lgq157q2bwjwq_jd18xm0000gq/T/f48b4761-8eff-4171-8857-edd5de0b3887 /private/var/folders/gx/kvm0lgq157q2bwjwq_jd18xm0000gq/T/5d01229f-0c79-4d8f-8e3b-a1398c1dc551
    timedOut: false
```

**Comment by austintude** [comment]
Yes, I have also had this issue but with zopflipng-bin. It crashes both npm run build AND npm run bundle. A work around I have just discovered is to place the image files in the root image folder (not the dev folder) and then the build and bundle actions WORK!

These were the errors thrown for me:
`[10:56:11] 'images' errored after 550 ms
[10:56:11] Error in plugin "gulp-image"
Message:
    Command failed: /Users/danielbisett/Documents/PushingLIMBits/wp-content/themes/pushing/node_modules/zopflipng-bin/vendor/zopflipng -y --lossy_8bit --lossy_transparent /private/var/folders/f2/zq9dry3n2fg8b6yqrfjxd2100000gn/T/c39d60b7-0b53-4f28-a1ec-5f5ac6e47704 /private/var/folders/f2/zq9dry3n2fg8b6yqrfjxd2100000gn/T/c280dcd6-1406-43d9-9b4f-042e31a7ef63

Optimizing /private/var/folders/f2/zq9dry3n2fg8b6yqrfjxd2100000gn/T/c39d60b7-0b53-4f28-a1ec-5f5ac6e47704
Encoding error 82: color conversion to palette requested while a color isn't in palette
Decoding error 82: color conversion to palette requested while a color isn't in palette
There was an error


Details:
    code: 1
    killed: false
    stdout: Optimizing /private/var/folders/f2/zq9dry3n2fg8b6yqrfjxd2100000gn/T/c39d60b7-0b53-4f28-a1ec-5f5ac6e47704
Encoding error 82: color conversion to palette requested while a color isn't in palette
Decoding error 82: color conversion to palette requested while a color isn't in palette
There was an error


    stderr: 
    failed: true
    signal: null
    cmd: /Users/danielbisett/Documents/PushingLIMBits/wp-content/themes/pushing/node_modules/zopflipng-bin/vendor/zopflipng -y --lossy_8bit --lossy_transparent /private/var/folders/f2/zq9dry3n2fg8b6yqrfjxd2100000gn/T/c39d60b7-0b53-4f28-a1ec-5f5ac6e47704 /private/var/folders/f2/zq9dry3n2fg8b6yqrfjxd2100000gn/T/c280dcd6-1406-43d9-9b4f-042e31a7ef63
    timedOut: false
    domainEmitter: [object Object]
    domain: [object Object]
    domainThrown: false`

**Comment by ataylorme** [comment]
@ammarjabakji and @austintude are you working with WP Rig version `1.0` or `2.0`?

The `dev` folder makes me things `1.0`. I don't believe this issue exists in `2.0`.

**Comment by ammarjabakji** [comment]
Hello Antaylorme,
Oops, I wasn't aware of that. Yes, I am working with `1.0`. 
I'll reinstall the lates version.
Thanks.

**Comment by ataylorme** [comment]
@ammarjabakji please be aware that `2.0` is still in development (beta) and is a complete re-write with breaking changes (we are following [semantic versioning](https://semver.org/)). You can't just upgrade without significantly reworking parts of your code.

I would recommend trying `2.0` on a new project rather than converting an existing one first.

**Comment by ammarjabakji** [comment]
@ataylorme I haven't done much in my project so it is fine for me to start over.
Many thanks for your efforts.

**Comment by mor10** [comment]
fixed in `v2.0`. Closing.
